CHANGES

```diff
@@ -46,6 +46,9 @@ Upcoming release: BREAKING
 * aarch32 VM fault messages now deliver original (untranslated) faulting IP in a hypervisor context, matching
   aarch64 behaviour.
 * Correct the minimum size of a scheduling context. This changes the value of `seL4_MinSchedContextBits`.
+* Changed how `gen_config.h` files are generated. Previously, they were generated at CMake configure time. Now, they
+  are generated at build time. For example, to manually build the kernel `gen_config.h` file after running `cmake`,
+  run `ninja gen_config/kernel/gen_config.h`.
```

Comment:
I wonder if it's worth mentioning that it's part of the generic `ninja kernel_Gen` now.

Reply:
I've added mention of both `${prefix}_Gen` and `${prefix}_Config`.

Reply:
I've removed mention of `${prefix}_Config`.